client/src/components/SideDrawer/index.tsx

```diff
@@ -242,6 +252,7 @@ const SideDrawer = () => {
                             >
                                 Submit
                             </Button>
+                            {jobError && <Alert sx={{mt: 1}} severity="error">All fields must be filled out</Alert>}
```

Comment:
If I try to add a job when I have no current cycle, it correctly displays an alert (because that's an invalid move), but "All fields must be filled out" is def not the correct message to display. Either make the error message even more generic, or make specific messages for the input issue